Dynatrace/dynatrace-operator
# Store initial connect retry value in curl config file

# Description

The value from the `feature.dynatrace.com/oneagent-initial-connect-retry-ms` feature flag should be passed to a file instead of an environment variable.
This file is now generated by the standalone runner and mounted by the webhook.

## How can this be tested?

* Create a test-namespace with test apps
* Deploy the operator 
* Apply a Dynakube with the mentioned flag set
* Restart the test-apps as soon as the operator deployment is ready
* The test app pods should have the `curl_options.conf` file mounted to `/var/lib/dynatrace/oneagent/agent/customkeys/curl_options.conf`
* Exec into one of the pods
* The log at `/opt/dynatrace/oneagent-paas/log/nginx/ruxitagent_nginx_myapp-__startup_1.0.log` should contain the line `Applied configuration from file: /var/lib/dynatrace/oneagent/agent/customkeys/curl_options.conf`

## Checklist
